src/content/docs/it/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,423 @@
+</FileTree>
+
+Puoi controllare questa posizione con la [nuova opzione di configurazione `build.assets`](/it/reference/configuration-reference/#buildassets).
+
+#### Cosa devo fare?
+
+Aggiorna la configurazione della tua piattaforma di distribuzione se quest'ultima si basa sulla posizione degli assets.
+
+### Cambiato: Configurazione dei plugin di markdown
+
+#### Rimosso: `extendDefaultPlugins`
+
+Nella v1.x, Astro usava `markdown.extendDefaultPlugins` per riattivare i plugin predefiniti di Astro quando venivano aggiunti dei propri plugin di Markdown.
+
+Astro v2.0 rimuove questa opzione completamente perchè questo comportamento ora è predefinito.
+
+L'applicazione di plug-in remark e rehype nella configurazione di Markdown **non disabilita più i plug-in predefiniti di Astro**. GitHub-Flavored Markdown e Smartypants ora vengono applicati indipendentemente dalla configurazione di `remarkPlugins` o `rehypePlugins` personalizzati.
+
+##### Cosa devo fare?
+
+Rimuovi `extendDefaultPlugins` nella tua configurazione. Ora questo è il comportamento predefinito di Astro nella v2.0, e puoi eliminare questa riga senza nessuna sostituzione.
+
+```js del={6}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  markdown: {
+    extendDefaultPlugins,
+  }
+});
+```
+
+#### Aggiunto: `gfm` e `smartypants`
+
+Nella v1.x, potevi scegliere se disabilitare entrambi i plugin di Markdown di Astro predefiniti (GitHub-Flavored Markdown e SmartyPants) impostando `markdown.extendDefaultPlugins: false`.
+
+Astro v2.0 sostituisce `markdown.extendDefaultPlugins: false` con delle opzioni booleane separate per controllare indipendentemente ognuno dei plugin Markdown di Astro. Questi sono abilitati di default e possono essere impostati su `false` in modo indipendente.
+
+##### Cosa devo fare?
+
+Rimuovi `extendDefaultPlugins: false` e aggiungi dei flag per disabilitare ciascun plugin indipendentemente.
+
+- `markdown.gfm: false` disables GitHub-Flavored Markdown
+- `markdown.smartypants: false` disables SmartyPants
+
+```js del={6} ins={7-8}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  markdown: {
+    extendDefaultPlugins: false,
+    smartypants: false,
+    gfm: false,
+  }
+});
+```
+
+### Cambiato: Configurazione del plugin MDX
+
+#### Sostituito: `extendPlugins` cambiato con `extendMarkdownConfig`
+
+Nella v1.x, l'opzione `extendPlugins` dell'integrazione MDX gestiva come i tuoi file MDX dovevano ereditare le configurazioni di Markdown: tutte le tue configurazioni di Markdown (`markdown`), o solamente i plugin predefiniti di Astro (`default`).
+
+Astro v2.0 sostituisce questo comportamento controllato da `mdx.extendPlugins` con tre nuove opzioni, configurabili indipendentemente, che sono impostate a `true`:
+
+- **[`mdx.extendMarkdownConfig`](/it/guides/integrations-guide/mdx/#extendmarkdownconfig)** to inherit all or none of your Markdown configuration
```

Comment:
**issue:** These 3 lines are still in English, e.g. `to inherit`.
I would suggest:
Line 184: `per ereditare tutte, o nessuna, le tue configurazioni Markdown`
Line 185: `per abilitare o disabilitare GitHub-Flavored Markdown nei file MDX`
Line 186: `per abilitare o disabilitare SmartyPants nei file MDX`